tools/psyq-obj-parser/psyq-obj-parser.cc

```diff
@@ -531,7 +533,8 @@ std::unique_ptr<PsyqLnkFile> PsyqLnkFile::parse(PCSX::File* file, bool verbose)
     return nullptr;
 }
 
-std::unique_ptr<PsyqLnkFile::Expression> PsyqLnkFile::Expression::parse(PCSX::File* file, bool verbose, int level) {
+std::unique_ptr<PsyqLnkFile::Expression> PsyqLnkFile::Expression::parse(std::unique_ptr<PCSX::File>& file, bool verbose,
```

Comment:
I think all of these `file` params should just be `File& file` as `unique_ptr` is forcing a heap allocation for the callers even if they don't need to heap allocate. Also this param seems like it can never be `nullptr` since it is never checked